data/json/items/magazine/38super.json

```
@@ -0,0 +1,18 @@
+[
+  {
+    "id": "m1911mag_10rd_38super",
+    "type": "MAGAZINE",
+    "name": "M1911 magazine .38 Super",
```

```suggestion
    "name": "M1911 .38 Super magazine",
```
For consistency with every other magazines (except for P226 .357 SIG magazine, this could be renamed too btw).